pmm-app/src/pmm-settings/Parts/Settings/Settings.tsx

```diff
@@ -69,6 +69,25 @@ export const getMetricsResolutionValues = metricsResolutions => {
   return metricsIndex;
 };
 
+const SECONDS = 60;
```

Comment:
It should be all in seconds, because that's what the API is looking for.
MINUTE  = 60;
HOUR = MINUTE * 60;
DAY = HOURS * 24;
Then the rest will make sense.